k8s/aat/common/divorce/div-rfe.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: divorce
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.nodejs: glob:prod-*
+    flux.weave.works/tag.nodejs: glob:pr-766-*
```

Comment:
why?

Reply:
I've updated some logging around PFE redirects and I want to see this on azure. The only way I can think of to test this is to point PFE demo to my PR branch and run through the journey on the frontend unless you know of another way?